lib/Transforms/Instrumentation/SimpleMemoryCheck.cc

```diff
@@ -30,12 +30,6 @@ static llvm::cl::opt<bool>
                   llvm::cl::desc("Print Simple Memory Check statistics"),
                   llvm::cl::init(false));
 
-static llvm::cl::opt<bool>
```

Comment:
@nhamlv-55 : did you intentionally change this file? Looks like it should not be part of the PR